gitlab-runner-mock/src/lib.rs

```diff
@@ -98,6 +100,10 @@ impl GitlabRunnerMock {
         &self.inner.runner_token
     }
 
+    pub fn system_id(&self) -> &str {
```

Comment:
I just realized that this should have been runner_system_id(). The fact that is wasn't caught during compilation hints that the mock code hasn't been compiled. I'm not sure what I did wrong though.

Reply:
From CI it looks like cargo test fails for a few other reasons as well :)